components/ModEditSubmissionTopbar.vue

```diff
@@ -47,13 +66,21 @@
 </template>
 
 <script setup lang="ts">
-import { ref, type Ref } from 'vue'
+import { ref, type Ref, watch, computed, onUnmounted } from 'vue'
+import { useI18n } from 'vue-i18n'
 import SVGCopyContent from '~/assets/icons/content-copy.svg'
 import SVGSuccessCheckMark from '~/assets/icons/checkmark-square.svg'
+import SVGCautionSign from '~/assets/icons/caution-sign.svg'
 import { useModerationSubmissionsStore } from '~/stores/moderationSubmissionsStore'
+import { startCountdownTimer, type CountdownCallback } from '~/utils/countdownTimer'
```

Comment:
uses a created timer that is in a separate file in case we need timers in the future that can use a callback to set other variables in case we need to set states. This one will use a callback that will change a local state of number of seconds and if the retry message should be shown. It is translatable